package.json

```diff
@@ -108,7 +107,7 @@
     "zxcvbn": "^4.4.2"
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": "8.* || >= 10.*"
```

Comment:
Frontend doesn't support or test against node 8, 10 is required.

Reply:
fixed.